fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneAutoCompletedMatchesTest.java

```diff
@@ -63,6 +63,21 @@ void searchForQua(String text, int numAdditionalTokens, List<String> expected) t
         assertComputeAllMatches("qua", Collections.emptyList(), ImmutableSet.of("qua"), text, numAdditionalTokens, expected);
     }
 
+    static Stream<Arguments> searchForCjk() {
+        return ImmutableList.of(
+                Arguments.of("世上无难事", "\"上无\"", Arrays.asList("上", "无"), null, 0, ImmutableList.of("上无")),
+                Arguments.of("世上无难事", "\"上无\"", Arrays.asList("上", "无"), null, 2, ImmutableList.of("上无难事")),
+                Arguments.of("世の中に難しいことなんてないよ", "\"の中に\"", Arrays.asList("の", "中", "に"), null, 3, ImmutableList.of("の中に難しい")),
+                Arguments.of("世ノ中ニシイコトナンテナ", "\"シイコ\"", Arrays.asList("シ", "イ", "コ"), null, 3, ImmutableList.of("シイコトナン")),
+                Arguments.of("세상에 어려운 일은 없다", "\"어려\"", Arrays.asList("어", "려"), null, 2, ImmutableList.of("어려운 일"))).stream();
```

Comment:
I don't have strongly-held opinions about the choice among `List.of`, `ImmutableList.of`, and `Arrays.asList`, but is there a reason to use different ones for the two `List<String>` arguments? It looks like some of the old code did that as well.